src/screens/Profile/AddEditFavorite/index.tsx

```diff
@@ -148,66 +137,72 @@ export default function AddEditFavorite({navigation, route}: AddEditProps) {
       />
 
       <View style={css.innerContainer}>
-        <Input
-          label="Sted"
-          value={location?.label}
-          placeholder="Søk etter adresse eller stoppested"
-          onFocus={() =>
-            navigation.navigate('LocationSearch', {
-              callerRouteName: AddEditRouteNameStatic,
-              callerRouteParam: 'searchLocation',
-              label: 'Sted',
-              favoriteChipTypes: ['location', 'map'],
-              initialLocation: location,
-            })
-          }
-          autoCorrect={false}
-          autoCompleteType="off"
-        />
-
-        <Input
-          label="Navn"
-          onChangeText={setName}
-          value={name}
-          editable
-          autoCapitalize="sentences"
-          accessibilityHint="Navn for favoritten"
-          placeholder="Legg til navn"
-        />
-
-        <InputGroup title="Ikon" boxStyle={{width: 124}}>
-          <SymbolPicker onPress={openEmojiPopup} value={emoji} />
-        </InputGroup>
-
-        <View style={css.line} />
+        <Input.Group withPadding>
+          <Input.Button
+            label="Sted"
+            value={location?.label}
+            placeholder="Søk etter adresse eller stoppested"
+            onPress={() =>
+              navigation.navigate('LocationSearch', {
+                callerRouteName: AddEditRouteNameStatic,
+                callerRouteParam: 'searchLocation',
+                label: 'Sted',
+                favoriteChipTypes: ['location', 'map'],
+                initialLocation: location,
+              })
+            }
+          />
+        </Input.Group>
+
+        <Input.Group withPadding>
+          <Input.Text
+            label="Navn"
+            onChangeText={setName}
+            value={name}
+            editable
+            autoCapitalize="sentences"
+            accessibilityHint="Navn for favoritten"
+            placeholder="Legg til navn"
+          />
+        </Input.Group>
+
+        <Input.Group withPadding>
+          <Input.Button
+            onPress={openEmojiPopup}
+            label="Ikon"
+            icon="expand-more"
+            type="inline"
+            value={
+              !emoji ? (
+                <ThemeIcon svg={MapPointPin} />
+              ) : (
+                <ThemeText type="body">{emoji}</ThemeText>
+              )
+            }
+          />
+        </Input.Group>
+      </View>
 
+      <ButtonGroup>
         <Button
           onPress={save}
           mode="primary"
-          icon={editItem ? Confirm : Add}
+          iconPosition="right"
+          icon={SvgConfirm}
           disabled={!hasSelectedValues}
```

Comment:
I know this button was disabled before as well. But validating, and helping the user correct their mistake may be preferable instead of disabling the button. We should consider if it's a good idea to do so.
In voiceover mode, this button has no indication of being disabled. It's accessible and "clickable"

Reply:
I updated this now and added error message flow instead, also using the accessibility reader component.

Reply:
I love it! 
A really nice improvement would be to focus the first field with missing/faulty input. But should not block this PR from being approved :+1: